packages/reactivity/src/collectionHandlers.ts

```diff
@@ -173,6 +173,24 @@ function createForEach(isReadonly: boolean, isShallow: boolean) {
   }
 }
 
+function createSetProtoMethod(method: TriggerOpTypes) {
+  return function (this: SetTypes, value: unknown, _isShallow = false) {
```

Comment:
Is anything passing the `_isShallow` flag? Is it actually needed?

Reply:
This is really not necessary